src/yoroi-wallets/cardano/transactionManager.ts

```diff
@@ -0,0 +1,62 @@
+import {Storage} from '../storage'
+import {BackendConfig} from '../types'
+import {TransactionCache} from './shelley'
+
+export const makeTransactionManager = async (storage: Storage) => {
+  const transactionCache = await TransactionCache.create(storage.join('txs/'))
+  const memosManager = await makeMemosManager(storage.join('memos/'))
+
+  return {
+    // transactionCache api
+    get transactions() {
```

Comment:
do we need these `get`s? I'm wondering why TS is not complaining

Reply:
`memo` getter provides the memos updated, if not just provide the memos gotten when the instance was created

Reply:
Should i refactor it ? thanks!

Reply:
I think you can just remove all `get` and add `as const` in the return.